solidity/contracts/AugurConstantProductMarket.sol

```diff
@@ -54,30 +53,30 @@ contract AugurConstantProduct is ERC20 {
 		// CONSIDER: selling complete sets incurs Augur fees, maybe we should let the user sell the sets themselves if they want to pay the fee?
 	}
 
-	function enterPosition(uint256 amountInDai, bool buyYes) external returns (uint256) {
+	function enterPosition(uint256 amountInDai, bool buyYes) external {
 		(uint256 poolInvalid, uint256 poolNo, uint256 poolYes) = shareBalances(address(this));
-		uint256 setsToBuy = amountInDai.div(numTicks);
+		uint256 setsToBuy = amountInDai / numTicks;
 
 		// simulate the user buying complete sets directly from the exchange
 		uint256 invalidToUser = setsToBuy;
 		uint256 noToUser = setsToBuy;
 		uint256 yesToUser = setsToBuy;
-		poolInvalid = poolInvalid.sub(invalidToUser, "AugurCP: The pool doesn't have enough INVALID tokens to fulfill the request.");
-		poolNo = poolNo.sub(noToUser, "AugurCP: The pool doesn't have enough NO tokens to fulfill the request.");
-		poolYes = poolYes.sub(yesToUser, "AugurCP: The pool doesn't have enough YES tokens to fulfill the request.");
+		poolInvalid = poolInvalid - invalidToUser;
+		poolNo = poolNo - noToUser;
+		poolYes = poolYes - yesToUser;
```

Comment:
We should probably do a `require(b>a, message)` for each of these so we get a clear/clean error message.